perun-rpc/src/main/java/cz/metacentrum/perun/rpc/methods/ConsentsManagerMethod.java

```diff
@@ -234,8 +234,8 @@ public Void call(ApiCaller ac, Deserializer params) throws PerunException {
 	/*#
 	 * Returns Service.
```

Comment:
You can remove this part and @ return parameter, also in the evaluateConsentsForConsentHub, since it returns nothing.